### PR TITLE
eliminating conversion of column names to lower case

### LINKED ISSUE #1208: Not able to retrieve row via REST with uppercase column name
Given a table created as

```sh
curl -L -X POST 'http://localhost:8082/v2/schemas/keyspaces/testks/tables' \
-H 'X-Cassandra-Token: <TOKEN>' \
-H 'Content-Type: application/json' \
--data-raw '{
    "name": "foo",
    "columnDefinitions": [
        {
            "name": "ID",
            "typeDefinition": "uuid"
        },
        {
            "name": "Firstname",
            "typeDefinition": "text"
        },
        {
            "name": "Lastname",
            "typeDefinition": "text"
        }
    ],
    "primaryKey": {
        "partitionKey": [
            "ID"
        ]
    },
    "ifNotExists": true
}'
```

I inserted a row with 

```sh
curl -L -X POST 'http://localhost:8082/v2/keyspaces/testks/foo' \
-H 'X-Cassandra-Token: <TOKEN>' \
-H 'Content-Type: application/json' \
--data-raw '{
    "ID": "af2603d2-8c03-11eb-a03f-0ada685e0000",
    "Firstname": "John",
    "Lastname": "Doe"
}'
```

But when I tried to retrieve the row it failed with the following error

```sh
curl -L -X GET 'http://localhost:8082/v2/keyspaces/testks/foo/af2603d2-8c03-11eb-a03f-0ada685e0000' \
-H 'X-Cassandra-Token: <TOKEN>'
{"description":"Bad request: Cannot find column id in table testks.foo","code":400}
```

It looks like the problem is we're always lowercasing the name here instead of preserving
https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/resources/Converters.java#L186

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Fixes #1208
**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated

### CODE REVIEW

File: restapi/src/main/java/io/stargate/web/resources/Converters.java [hunk 1]
```diff
@@ -183,7 +183,7 @@ public static BuiltCondition idToWhere(String val, String name, Table tableData)
       value = toCqlValue(type, val);
     }
 
-    return BuiltCondition.of(name.toLowerCase(), Predicate.EQ, value);
+    return BuiltCondition.of(name, Predicate.EQ, value);
```
Comment (dougwettlaufer) [inline restapi/src/main/java/io/stargate/web/resources/Converters.java:186]: Can we add some integration tests for this?

Comment (ivansenic) [reply restapi/src/main/java/io/stargate/web/resources/Converters.java:186]: Furthermore, is this backward compatible or would we break somebody? Also, are there other places where `.toLowerCase()` is used?

**Review by ivansenic — APPROVED** [review]
+1 for the INT test, seems CI fails due to the formatting?

File: restapi/src/main/java/io/stargate/web/resources/Converters.java (hunk 1 above)
Comment (jeffreyscarpenter) [reply restapi/src/main/java/io/stargate/web/resources/Converters.java:186]: I searched and PR fixes two places where `.toLowerCase()` is used.

I added an integration test that tests creating and accessing tables with mixed case table name and column names.

**Pull request merged**